lua/autorun/client/cl_mcsp.lua

```diff
@@ -0,0 +1,38 @@
+local MCSPmodel = "models/milaco/minecraft_pm/minecraft_pm.mdl"
+
+hook.Add("PrePlayerDraw", "MCSPRenderSkin", function(ply) --Material ID 0 is the hat layer, ID 1 is the base layer
+	if IsValid(ply) and ply:GetModel() == MCSPmodel and ply:IsPlayer() then
+		render.MaterialOverrideByIndex(1, ImgurMaterial(ply:GetNWString("MCSPSkinURL", false), ply, ply:GetPos(), false, "VertexLitGeneric", {
```

Comment:
i'm not sure what will happen if the NWString("MCSPSkinURL") is unset and returns false, can you check for that in the first line of this function and not override if so?